priv/emqx.schema

```diff
@@ -387,11 +387,16 @@ end}.
   {datatype, {enum, [off, file, console, both]}}
 ]}.
 
-{mapping, "log.level", "kernel.logger_level", [
+{mapping, "log.level", "kernel.logger", [
   {default, error},
   {datatype, {enum, [debug, info, notice, warning, error, critical, alert, emergency, all]}}
 ]}.
 
+{mapping, "log.primary_level", "emqx.primary_log_level", [
+   {default, error},
+   {datatype, {enum, [debug, info, notice, warning, error, critical, alert, emergency, all]}}
+]}.
```

Comment:
What is the difference between log level and primary log level

Reply:
http://erlang.org/doc/apps/kernel/logger_chapter.html#overview